.env.example

```
@@ -0,0 +1,23 @@
+NODE_ENV=development # or 'production'
+PORT=3000
+LISTEN_ADDRESS=0.0.0.0
+
+SESSION_SECRET=not-secure-change-me # an UUID or `openssl rand -hex 32` or something similar
+SESSION_MAX_AGE= 604800000 # TTL in ms. example is equal to 7 weeks (1000 * 60 * 60 * 24 * 7)
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)